src/sass/components/_tour.scss

```diff
@@ -29,3 +29,37 @@
   }
 }
 /* stylelint-enable declaration-no-important */
+
+// Override background image
+.slds-popover_walkthrough .slds-popover__header {
+  background-image: url('~img/popover-header.png');
+}
+
+.popover-wrapper {
+  position: absolute;
+  top: -10px;
+
+  .slds-page-header__title & {
+    top: unset;
+  }
+
+  .page-title & {
+    top: 25px;
+  }
+}
+
+.popover-button {
```

Comment:
@dvdherron  How would you feel about something like this to better match @SondraE's mock up? I know it overrides some of the LDS design though so there's that to weigh:

```
.popover-button {
  color: $palette-violet-65;
  background: $color-background-light;
  border: 1px solid $palette-violet-90;
  border-radius: 100%;
  box-shadow: 0 0 0 1px $palette-violet-65;

  .tour-alert & {
    fill: currentColor;
  }

  &:hover,
  &:focus {
    // first line is the SLDS default button shadow, second is our addition
    box-shadow: var(--sds-c-button-shadow-focus, 0 0 3px $color-brand-dark
    ),
      0 0 0 1px $color-text-link-focus;
  }
}
```

In addition to the visual match, this code also adds a white background to the `i` character cut out which helps in the alert bar and anywhere the icon is over something with a background or border behind it.

Reply:
I think this works great and makes these buttons look even better 👍🏽

Reply:
@dvdherron @stacyk I like the white background, but (in FF on Mac at least) these now look much bigger -- I would say too big, but I guess that's a matter of preference. (If we really want them this big, I'd vote for a bit more spacing between them and whatever they're highlighting.)

The background shadow is also off-centered, and is a slightly cut-off circle which looks unintentional (it is fixed on hover).

![Screen Shot 2021-04-23 at 10 50 15 AM](https://user-images.githubusercontent.com/552316/115889229-c15e7080-a421-11eb-90d9-2414af0fa7ac.png)

Reply:
(It now looks much better to me. Same questions remain about overriding SLDS, but I'm okay with this as-is. 👍)